ubuntudesign/vanilla-brochure-theme
# Add jekyll to view examples

## Done

Add example jekyll site to mirror vanilla-framework

## QA

- Run `rm -rf node_modules`
- Run `npm i`
- Run `bundle install`
- Run `gulp jekyll`
- Example site should be at: `http://localhost:4001`
